lib/color_guard/store/stack.rb

```diff
@@ -19,6 +19,10 @@ def find(feature_name)
         end
       end
 
+      def all
+        @stores.map { |store| store.all }
+      end
+
```

Comment:
As mentioned earlier I'm a bit concerned that this has a different return type from `#all` on the other stores. The point of the stack store was to have it indistinguishable from other stores at the interface level.

Not sure quite how to solve this right now but we can talk through it later to come up with a solution for both cases